backend_new/prisma/schema.prisma

```diff
@@ -0,0 +1,1025 @@
+  applicationsAlternateAddress Address?                      @relation("applications_alternate_address", fields: [alternateAddressId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+  userAccounts                 UserAccounts?                 @relation(fields: [userId], references: [id], onUpdate: NoAction)
+  applicationsMailingAddress   Address?                      @relation("applications_mailing_address", fields: [mailingAddressId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+  listings                     listings?                     @relation(fields: [listingId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+  demographics                 Demographics?                 @relation(fields: [demographicsId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+  preferredUnitTypes           UnitTypes[]
+  householdMember              HouseholdMember[]
+
+  @@unique([listingId, confirmationCode])
+  @@index([listingId])
+  @@map("applications")
+}
+
+// Note: [file_id, label] formerly max length 128
+model Assets {
+  id                            String              @id() @default(dbgenerated("uuid_generate_v4()")) @db.Uuid
+  createdAt                     DateTime            @default(now()) @map("created_at") @db.Timestamp(6)
+  updatedAt                     DateTime            @updatedAt @map("updated_at") @db.Timestamp(6)
+  fileId                        String              @map("file_id")
+  label                         String
+  listingEvents                 ListingEvents[]
+  listingImages                 ListingImages[]
+  buildingSelectionCriteriaFile listings[]          @relation("building_selection_criteria_file")
+  listingsResult                listings[]          @relation("listings_result")
+  paperApplications             PaperApplications[]
+
+  @@map("assets")
+}
+
+// Note: [name] formerly max length 64
+model CronJob {
+  id          String    @id() @default(dbgenerated("uuid_generate_v4()")) @db.Uuid
+  name        String?
+  lastRunDate DateTime? @map("last_run_date") @db.Timestamptz(6)
+  createdAt   DateTime  @default(now()) @map("created_at") @db.Timestamp(6)
+  updatedAt   DateTime  @updatedAt @map("updated_at") @db.Timestamp(6)
+
+  @@map("cron_job")
+}
+
+// Note: [ethnicity, gender, sexual_orientation, how_did_you_hear] formerly max length 64
+// Note: [how_did_you_hear, race] formerly max array length 64
+model Demographics {
+  id                String        @id() @default(dbgenerated("uuid_generate_v4()")) @db.Uuid
+  createdAt         DateTime      @default(now()) @map("created_at") @db.Timestamp(6)
+  updatedAt         DateTime      @updatedAt @map("updated_at") @db.Timestamp(6)
+  ethnicity         String?
+  gender            String?
+  sexualOrientation String?       @map("sexual_orientation")
+  howDidYouHear     String[]      @map("how_did_you_hear")
+  race              String[]
```

Comment:
Should be optional

Reply:
this is a difference between prisma and typeorm 

in typeorm arrays can be nulled out
in prisma optional lists are not nullable

this would just become the empty [] instead of null for us